.github/workflows/stable-release.yaml

```diff
@@ -0,0 +1,21 @@
+name: stable-release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: yarn
+      - run: yarn test
+      - run: yarn build
+      - run: yarn version --no-git-tag-version --major --preid=stable --access=public
```

Comment:
```suggestion
```
no need to do this on stable release it should take the version from `package.json`